__tests__/help.ts

```diff
@@ -17,6 +22,16 @@ test('npx unimported --help', (done) => {
         --flow, -f    indicates if your code is annotated with flow types    [boolean]
         --update, -u  update the ignore-lists stored in .unimportedrc.json   [boolean]"
     `);
-    done();
-  });
+  done();
+});
+
+test('error npx unimported --help', async (done) => {
+  // wrong file name
+  try {
+    await exec('node ./bin/1unimported.js --help');
+  } catch (e) {
+    expect(e).not.toBe(null);
```

Comment:
The problem with an assertion in a `catch` block is that if the promise resolves successfully, this will never run and the test will pass.  This is why the promise or async/await assertions I linked in my previous comment can be more useful.

Reply:
Also, generally if your expecting an error, you'd assert the message to make sure some else didn't throw and it's just accidentally passing.